packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpMvsApi.ts

```diff
@@ -122,14 +125,20 @@ export class FtpMvsApi extends AbstractFtpApi implements ZoweExplorerApi.IMvs {
         try {
             connection = await this.ftpClient(this.checkedProfile());
             if (!connection) {
+                ZoweLogger.logImperativeMessage(result.commandResponse, MessageSeverityEnum.ERROR);
                 throw new Error(result.commandResponse);
             }
             // Save-Save with FTP requires loading the file first
             if (options.returnEtag && options.etag) {
                 const contentsTag = await this.getContentsTag(dataSetName);
                 if (contentsTag && contentsTag !== options.etag) {
                     // TODO: extension.ts should not check for zosmf errors.
-                    throw new Error("Save conflict. Please pull the latest content from mainfram first.");
+                    ZoweVsCodeExtension.showVsCodeMessage(
+                        "Save conflict. Please pull the latest content from mainframe first.",
+                        MessageSeverityEnum.ERROR,
+                        ZoweLogger
+                    );
+                    throw new Error();
```

Comment:
Should the error be populated in the event that someone may still use these APIs programmatically?
Otherwise, they will need to check for information passed to the showVsCodeMessage function prior to the error being thrown.

This may apply to other API calls.

Not a hard requirement though